users/api/http/endpoint.go

```diff
@@ -23,6 +23,56 @@ func registrationEndpoint(svc users.Service) endpoint.Endpoint {
 	}
 }
 
+// Password reset request endpoint.
+// When successful password reset link is generated.
+// Link is generated using MF_TOKEN_RESET_ENDPOINT env.
+// and value from Referer header for host.
+// {Referer}+{MF_TOKEN_RESET_ENDPOINT}+{token=TOKEN}
+// http://mainflux.com/reset-request?token=xxxxxxxxxxx.
+// Email with a link is being sent to the user.
+// When user clicks on a link it should get the ui with form to
+// enter new password, when form is submitted token and new password
+// must be sent as PUT request to 'password/reset' passwordResetEndpoint
```

Comment:
There is no need to mention UI here. In order to recover the password, issue the PUT request containing the token from the link and new password to the `password/reset`.